doc/develop/test/blackbox.rst

```diff
@@ -0,0 +1,180 @@
+.. _blackbox:
+
+======================
+Twister blackbox tests
+======================
+
+Basics
+======
+
+Twister blackbox tests are written in python, with auxiliary test data following
+whichever format it was in originally. They are wholly contained in the
+`scripts/tests/twister_blackbox/` directory inside zephyr and prepended with `test_`.
+
+Blackbox tests should not be aware of the internal twister code. Instead, they should
+call twister as user would and check the results.
+
+This README aims to explain the structure of a test file so the reader will be able
```

Comment:
now this guide is a part of Zephyr documentation, not just a README as it was.
Would be nice to have  README as well just with few examples of how to run these tests.